src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    private static final FigureSupplier FIGURE_SUPPLIER = new FigureSupplier();
```

Comment:
```suggestion
    private FigureSupplier figureSupplier = new FigureSupplier();
```

Reply:
I removed it in main() method, because I need to use only static value in main().